From 6bc5bcc6d003b9faee54450a0142c91ec9cb53ab Mon Sep 17 00:00:00 2001
From: kai-dev <kai-dev@email.com>
Date: Sat, 22 Aug 2020 20:51:21 +0800
Subject: [PATCH] err

---
 Android.mk | 38 +++++---------------------------------
 1 file changed, 5 insertions(+), 33 deletions(-)

diff --git a/Android.mk b/Android.mk
index f0a0eaa..6c8d5eb 100644
--- a/Android.mk
+++ b/Android.mk
@@ -177,15 +177,15 @@ endif
 $(LOCAL_INSTALLED_MODULE) : $(LOCAL_BUILT_MODULE)
 	@echo Install dir: $(TARGET_OUT)/b2g
 	rm -rf $(filter-out $(addprefix $(TARGET_OUT)/b2g/,$(PRESERVE_DIRS)),$(wildcard $(TARGET_OUT)/b2g/*))
-	cd $(TARGET_OUT) && tar xvfz $(abspath $<)
+	:
 ifneq ($(EXPORT_DEVICE_PREFS),)
-	cp -n $(EXPORT_DEVICE_PREFS)/*.js $(TARGET_OUT)/b2g/defaults/pref/
+	:
 endif
 ifneq ($(EXPORT_DEVICE_USER_BUILD_PREFS),)
-	cp -n $(EXPORT_DEVICE_USER_BUILD_PREFS) $(TARGET_OUT)/b2g/defaults/pref/
+	:
 endif
 ifneq ($(EXPORT_BUILD_PREFS),)
-	cp -n $(EXPORT_BUILD_PREFS) $(TARGET_OUT)/b2g/defaults/pref/
+	:
 endif
 
 GECKO_LIB_DEPS := \
@@ -250,35 +250,7 @@ endif
 
 .PHONY: $(LOCAL_BUILT_MODULE)
 $(LOCAL_BUILT_MODULE): $(TARGET_CRTBEGIN_DYNAMIC_O) $(TARGET_CRTEND_O) $(addprefix $(TARGET_OUT_SHARED_LIBRARIES)/,$(GECKO_LIB_DEPS)) $(GECKO_LIB_STATIC)
-ifeq ($(USE_PREBUILT_B2G),1)
-	@echo -e "\033[0;33m ==== Use prebuilt gecko ==== \033[0m";
-	mkdir -p $(@D) && cp $(abspath $(PREFERRED_B2G)) $@
-else
-	echo "export GECKO_OBJDIR=$(abspath $(GECKO_OBJDIR))"; \
-	echo "export GONK_PRODUCT_NAME=$(TARGET_DEVICE)"; \
-	echo "export GONK_PATH=$(abspath .)"; \
-	echo "export PLATFORM_VERSION=$(PLATFORM_SDK_VERSION)"; \
-	echo "export TARGET_ARCH=$(TARGET_ARCH)"; \
-	echo "export TARGET_ARCH_VARIANT=$(TARGET_ARCH_VARIANT)"; \
-	echo "export TARGET_CPU_VARIANT=$(TARGET_CPU_VARIANT)"; \
-	echo "export PRODUCT_MANUFACTURER=$(PRODUCT_MANUFACTURER)"; \
-	echo "export MOZ_DISABLE_LTO=$(MOZ_DISABLE_LTO)"; \
-	echo "export HOST_OS=$(HOST_OS)";	\
-	unset CC_WRAPPER && unset CXX_WRAPPER && \
-	export GECKO_OBJDIR="$(abspath $(GECKO_OBJDIR))" && \
-	export GONK_PATH="$(abspath .)" && \
-	export GONK_PRODUCT_NAME="$(TARGET_DEVICE)" && \
-	export PLATFORM_VERSION="$(PLATFORM_SDK_VERSION)" && \
-	export TARGET_ARCH="$(TARGET_ARCH)" && \
-	export TARGET_ARCH_VARIANT="$(TARGET_ARCH_VARIANT)" && \
-	export TARGET_CPU_VARIANT="$(TARGET_CPU_VARIANT)" && \
-	export PRODUCT_MANUFACTURER="$(PRODUCT_MANUFACTURER)" && \
-	export MOZ_DISABLE_LTO="$(MOZ_DISABLE_LTO)" && \
-	export HOST_OS="$(HOST_OS)" && \
-	(cd $(GECKO_PATH) ; $(SHELL) build-b2g.sh) && \
-	(cd $(GECKO_PATH) ; $(SHELL) build-b2g.sh package) && \
-	mkdir -p $(@D) && cp $(GECKO_OBJDIR)/dist/b2g-*.tar.gz $@
-endif
+	:
 
 .PHONY: buildsymbols
 buildsymbols:
-- 
2.17.1

